src/app/select-trajectory/select-trajectory.page.ts

```diff
@@ -117,7 +120,14 @@ export class SelectTrajectoryPage implements OnInit {
         })
         modal.present()
         const { data: t } = await modal.onWillDismiss<TrajectoryMeta>()
-        if (t) this.router.navigate([`/trajectory/${t.type}/${t.id}`])
+        if (t) {
+          this.logfileService.log(
+            'User opens map view',
```

Comment:
This might be a bit misleading, even though the map view is what the user sees first. Maybe sth. like _Trajectory selected_ is better here?